README.rst

```diff
@@ -56,7 +56,7 @@ Testing
 -------
 
 Tests have been for python 2.7 and python 3 using ``pytest``. The unit
-tests do call the actual PurgoMalum production API in order to test
+tests do call the actual LibInsult's `production API <https://insult.mattbas.org/api/>`__ in order to test
```

Comment:
You need to do a rebase - you already made this change and I committed it.

So what I usually do is: a) switch to master, b) do a git pull to get the latest, c) create a new local branch, and then d) apply my new changes.  Since you've already made this commit you can do steps a, b, and c.  Then you can cherry-pick your commit from this branch you've already created (use `git cherry-pick`)

Reply:
Basically, what i'm saying is that this change is not needed.  So just remove it or do the steps above so that you don't need it anymore.